src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,30 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { user } = post;
```

Comment:
```suggestion
  const { user, body, title, comments } = post;
```
get all properties using destructuring assignment. BTW, it would be nice to do the same in other files